Code/GraphMol/Chirality.cpp

```diff
@@ -118,16 +118,13 @@ namespace RDKit{
       }
     }
 
-    void iterateCIPRanks(const ROMol &mol, DOUBLE_VECT &invars, UINT_VECT &ranks,bool seedWithInvars){
+    void iterateCIPRanks(const ROMol &mol, DOUBLE_VECT &invars,
```

Comment:
I'll remove this, this is part of another optimization roll up I should have removed from this commit.